jobs/env/ci-kubernetes-e2e-kubemark-common.env

```diff
@@ -12,3 +12,7 @@ SCHEDULER_TEST_ARGS=--profiling
 TEST_CLUSTER_RESYNC_PERIOD=--min-resync-period=12h
 # Reduce etcd compaction frequency to match production.
 KUBEMARK_ETCD_COMPACTION_INTERVAL_SEC=150
+
+# Use Taint based evictions to control hollow node recreation in case of
+# node VM restart.
```

Comment:
Could you add a link to the issue here for context?

Reply:
Done.